bin/sozo/src/commands/mod.rs

```diff
@@ -63,6 +65,8 @@ pub enum Commands {
     Model(Box<ModelArgs>),
     #[command(about = "Inspect events emitted by the world")]
     Events(Box<EventsArgs>),
+    #[command(about = "Verify contracts in walnut.dev - transactions debugger and simulator")]
+    WalnutVerify(Box<VerifyArgs>),
```

Comment:
@glihm I've changed it from "Verify" to "WalnutVerify". Please note that after this change calling walnut verification is calling `sozo walnut-verify`.

Reply:
If walnut is meant to be updated in the future and we may have something like `sozo walnut verify` or `sozo walnut debug`, then consider adding an other level.
If not, it's good like so. 👍 

And even for the commands, they can be coded into the walnut crate. And here, only imported and used if the feature is actually enabled.